lib/elixir/lib/protocol.ex

```diff
@@ -280,9 +280,11 @@ defmodule Protocol do
     type_args = :lists.map(fn _ -> quote(do: term) end, :lists.seq(2, arity))
     type_args = [quote(do: t) | type_args]
 
-    varify = fn pos -> Macro.var(String.to_atom("arg" <> Integer.to_string(pos)), __MODULE__) end
+    convert_variables = fn pos ->
```

Comment:
I think the plural is a bit confusing since the function only returns one var?
How about just `to_var`?

Reply:
Makes sense 👍